### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by Nocturnal42** [Fiona1729/py-rolldice#8]
When I install and attempt to run, using the usage example in the readme.md I get the following error.

```
>>> import rolldice
>>> result, explanation = rolldice.roll_dice('12d6 + 10')
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/rolldice/rolldice.py", line 161, in _eval
    handler = self.nodes[type(node)]
KeyError: <class '_ast.Num'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/rolldice/rolldice.py", line 855, in roll_dice
    final_result = parser.eval(''.join([str(x) for x in results])) #Call the parser to parse into one value
  File "/usr/local/lib/python3.7/dist-packages/rolldice/rolldice.py", line 151, in eval
    return self._eval(ast.parse(expr.strip()).body[0].value)
  File "/usr/local/lib/python3.7/dist-packages/rolldice/rolldice.py", line 165, in _eval
    return handler(node)
  File "/usr/local/lib/python3.7/dist-packages/rolldice/rolldice.py", line 196, in _eval_binop
    return self.operators[type(node.op)](self._eval(node.left),
  File "/usr/local/lib/python3.7/dist-packages/rolldice/rolldice.py", line 163, in _eval
    raise ValueError("Sorry, {0} is not available in this evaluator".format(type(node).__name__))
ValueError: Sorry, Num is not available in this evaluator

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/dist-packages/rolldice/rolldice.py", line 859, in roll_dice
    raise DiceOperatorException('Error parsing operators and or functions')
rolldice.rolldice.DiceOperatorException: Error parsing operators and or functions

```

**Comment by MajorTanya** [comment]
I remember the last update being a fix for Python 3.8 and then a PR to make the fix work with 3.7 and below, but there never was a version update so the 3.8+ exclusive version seems to be the distributed one right now. 

I see that you're using 3.7, is upgrading to 3.8 an option? Otherwise, pyrolldice v. 0.3.9 should work just fine.

**Comment by Nocturnal42** [comment]
Upgrading python is not really viable without also upgrading my distro, which I would rather not do. So I went with a third dirtier option. I directly edited the installed package and applied the patch from the PR to support 3.7.

Thanks for the prompt response.
